### ISSUE TITLE
Parameter doNotValidate throws error

### ISSUE THREAD

**Issue by Zefau** [Mik13/nuki-bridge-api#7]
Hi there,

thanks again for your effort.
I have implemented the `get` function now to retrieve a single nuki from a bridge based on its ID.

Using the second parameter `doNotValidate` fails though with the error
```
TypeError: bridge.get(...).then is not a function
```

My code:
```
bridge.get(nukiId, true).then(function(device)
```

https://github.com/Mik13/nuki-bridge-api/blob/aa99e37639f85ac9b491d0a6675dfce9a52a8ac8/lib/bridge.js#L86

**Comment by Zefau** [comment]
@Mik13 Any news on this? :)

**Comment by Mik13** [comment]
Sorry for the late response. Fixed in Version 1.3.1

Thanks for reporting and best regards
